src/persistence/api/views.py

```diff
@@ -180,3 +187,295 @@ def post(self, request):
             return Response(data, status=status.HTTP_403_FORBIDDEN)
         serializer.save()
         return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+
+class GenerateForkTokenView(GenericAPIView):
+    """
+    Generates a token for the fork/server and account identifier.
+    **Requires token in 'X-Character-Token' header.**
+    """
+
+    serializer_class = CharacterSerializer
+
+    def post(self, request):
+        server_id = request.data.get("fork_compatibility")
+        if not server_id:
+            return Response(
+                {"error": "Missing 'fork_compatibility' in request body."}, status=status.HTTP_400_BAD_REQUEST
+            )
+
+        user = request.user
+        if not hasattr(user, "unique_identifier"):
+            return Response(
+                {"error": "Authenticated user lacks a unique identifier."}, status=status.HTTP_500_INTERNAL_SERVER_ERROR
+            )
+
+        token_data = {
+            "server_id": server_id,
+            "uuid": str(user.unique_identifier),
```

Comment:
```suggestion
            "unique_identifier": user.unique_identifier,
```
let's use the same words everywhere so we always know what we are talking about